### ISSUE TITLE
Custom fields synchronisation during lead conversion

### ISSUE THREAD

**Issue by sadortun** [orocrm/crm#226]
Hello Artem,

When a custom entity field exist on a Lead and on an Account or Contact for example, is there a mecanism to set the related field value ?  

The mechanism exists for a defined set of fields, but can it be applied also to extended proprieties ?
https://github.com/orocrm/crm/blob/5a23255d0f15b5ae42e5aa07b3c0716201ae55ba/src/Oro/Bundle/SalesBundle/Provider/LeadToOpportunityProvider.php#L74 

Thanks,
Samuel

**Comment by mkudelya** [comment]
Hi @sadortun 

As i understood do you want to create two identical field in lead and contact and when you are converting lead to opportunity set value of this field to contact ? I have tested two variants of field, text and relation 'many-to-one', after converting i had contact with appropriate value. I just added my field to [properties](https://github.com/orocrm/crm/blob/5a23255d0f15b5ae42e5aa07b3c0716201ae55ba/src/Oro/Bundle/SalesBundle/Provider/LeadToOpportunityProvider.php#L62) key.

**Comment by sadortun** [comment]
Hi @mkdotam, Exact, i've also done this test. I think it would be a great feature to be able to add this mapping either via the Entity admin, or via configuration files, or by default.

Since this applies only during the lead conversion, I think they would not have any  unwanted effects.
- Only occur when a lead is converted to Opportunity (and related entities)
- Only occur if the target field is empty.

What do you think ?

**Comment by mkudelya** [comment]
@sadortun 

Thank you for the idea, we have added it to the backlog. I will notify you when we will prioritize it for implementation

**Comment by artml** [comment]
Yes, that's a neat idea and will give our system better customization options. Thank you Samuel!

**Comment by sadortun** [comment]
Great! THanks !

**Comment by sadortun** [comment]
@artml @mkudelya any update on this ?

**Comment by artml** [comment]
@sadortun nope. The task is still in our backlog but we didn't prioritize it.

**Comment by sadortun** [comment]
@artml Can you have a quick check if this should be implemented, if so, we might be selected to submit a PR. 

Do you see any side effects or pitfalls ?

**Comment by manowark** [comment]
Hi, @sadortun.
We will check your pull request in a short time. Thank you for the contribution.

**Comment by mbessolov** [comment]
_(internal reference CRM-6997)_
